site/documentation/content/api/event/index.md

```diff
@@ -130,3 +132,88 @@ The example below might be used by the MQTT adapter to indicate that a connectio
   }
 }
 ~~~
+
+
+## Event API for Kafka
+
+Hono can alternatively be configured to publish event messages to an Apache Kafka&reg; cluster instead of an AMQP Messaging Network. The Event API for Kafka is defined by means of Kafka messages.
+
+### Southbound Operations
+
+The following operation can be used by *Protocol Adapters* to send event messages received from devices to downstream consumers like *Business Applications*.
+
+#### Produce Event
+
+The protocol adapter writes messages to the tenant-specific topic `event-${tenant_id}` where `${tenant_id}` is the ID of the tenant that the client wants to upload event messages for.
+
+
+**Preconditions**
+
+1. Either the topic `event-${tenant_id}` exists, or the broker is configured to automatically create topics on demand.
+1. The client is authorized to write to the topic.
+1. The device for which the adapter wants to send event messages has been registered (see [Device Registration API]({{< relref "/api/device-registration" >}})).
+
+**Message Flow**
+
+Hono supports *AT LEAST ONCE* delivery of *Event* messages only. A client therefore MUST set *acks* to `all`. Either MUST *enable.idempotence* be set to `true` OR *max.in.flight.requests.per.connection* MUST be set to `1`.
```

Comment:
You mean having each configuration property explained in one place and then referencing to this where they are mentioned? I like the idea. But it needs to focus on _why_ and _how_ the property is relevant to Hono, because the Kafka documentation already does a great job in explaining each configuration option: https://kafka.apache.org/documentation/#producerconfigs
We need to be careful here, because, in contrast to the AMQP 1.0 specification, Kafka is not a standard and it is constantly evolving.